CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 2.5.0
+
+* **Experimental: [Playwright](/playwright) helper introduced**.
+
+> [Playwright](https://github.com/microsoft/playwright/) is an alternative to Puppeteer which works very similarly to it but adds cross-browser support with Firefox and Webkit. Until v1.0 Playwright API is not stable but we introduce it to CodeceptJS so you could try it.
+
+* [Puppeteer] Fixed basic auth support when running in multiple sessions. See #2178 by @ian-bartholomew
+* [Puppeteer] Fixed `waitForText` when there is no `body` element on page (redirect). See #2181 by @Vorobeyko
+* [Selenoid plugin] Fixed overriding current capabilities by adding deepMerge. Fixes #2183 by @koushikmohan1996
+* Added types for `Scenario.todo` by @Vorobeyko
+* Added `--colors` option to `run` and `run-multiple` so you force colored output in dockerized environment. See #2189 by @mirao
+
```

Comment:
Maybe we miss those #2166, #2164, #2169

Reply:
No they were referenced in previous release

Reply:
sorry, I misread the release notes :)